.travis.yml

```diff
@@ -2,12 +2,9 @@ os:
   - linux
 language: python
 python:
-  - "2.7"
   - "3.6"
   - "3.7"
   - "3.8"
   - "nightly"
 # command to run tests
-install:
-# Nothing needed
 script: python setup.py test
```

Comment:
Not really @agronholm's problem but [`setup.py test` is deprecated](https://setuptools.readthedocs.io/en/latest/setuptools.html#test-build-package-and-run-a-unittest-suite) and will be removed in the relatively near future (though I'm disturbed to find that there is no deprecation warning in the Travis CI logs for some reason).

Reply:
I'd be happy to overhaul the packaging and testing configuration but I figured it was beyond the scope of this PR.
For example, Travis only tests this on Linux even though there's Windows specific code.